virt/virtual_machines/virt-managing-configmaps-secrets-service-accounts.adoc

```diff
@@ -0,0 +1,31 @@
+[id="virt-managing-configmaps-secrets-service-accounts"]
+= Managing ConfigMaps, secrets, and service accounts in virtual machines
+include::modules/virt-document-attributes.adoc[]
+:context: virt-managing-configmaps-secrets-service-accounts
+toc::[]
+
+
+You can use secrets, ConfigMaps, and service accounts to pass configuration data to virtual machines (VMs). For example, you can:
```

Comment:
Nit: Not sure if it's worth the VM abbreviation as it's only used twice in the assembly here and then all other instances in the included modules are 'virtual machines'.